content/related_work.md

```diff
@@ -87,8 +90,56 @@ Referenced collections:
 - MAWILab
 - UMass Trace Repository
 
+
+### Are Public Intrusion Datasets Fit for Purpose Characterising the State of the Art in Intrusion Event Datasets (2020)
+
+[Link to Paper](https://doi.org/10.1016/j.cose.2020.102022)
+```
+Kenyon, A., Deka, L., & Elizondo, D. (2020). Are public intrusion datasets fit for purpose characterising the state of the art in intrusion event datasets. Computers & Security, 99, 102022.
+```
+
+The authors focus on analysing the - according to them - most widely used  public intrusion datasets, also noting that their relative scarcity is compounded by the lack of central registry of some sort.
+To alleviate this problem, and to investigate the current state of the art, they study 25 datasets as well as three collections, providing a small set of features (origin, anonymization, labeling, flow availability, attack types) and a description for each.
+They also enumerate a number of characteristic a dataset should fulfil in order to be suitable for IDS research, and conclude that, at the time of writing, there is no such dataset.
+A couple of datasets deemed noteworthy are highlighted.
```

Comment:
This sentence is confusing IMHO. Maybe more clearly "The authors recommend the UNB IDS datasets as well as the MAWI and CAIDA datasets since they fulfill most of the aforementioned characteristics."?